splat.pl

```diff
@@ -0,0 +1,953 @@
+	    }
+	}
+    }
+
+    print STDERR "SPLAT% Finished. \n";
+    $top->withdraw();
+    exit(0);
+}
+
+##***************************************************************************
+sub open_bin_output {
+##  Open output binary file if needed
+##***************************************************************************
+  eval "use FileHandle";
+  
+  @out_num = split(/,/ , $opt_out);
+  foreach (@out_num) {
+    $DAT_fh[$_] = new FileHandle "> acaslot$_.dat";
+    die "Couldn't open acaslot$_.dat\n" unless defined $DAT_fh[$_];
+    $DAT_fh[$_]->autoflush(1);
+  }
+}
+
+##***************************************************************************
+sub get_options {
+##***************************************************************************
+  $Usage = <<'USAGE';
+  Usage: splat.pl [-h] [-i tlm_file] [-image] [-limits <file>] [-logsc] 
+                  [-out <file>] [-page] [-pea (a|b)] [-raw] [-time <msec>] [-tctm]
+     -h     : Print help
+     -i     : Optional input telemetry file
+     -image : Show images (default, use -noimage otherwise)
+     -limits: Limits file (not implemented yet)
+     -logsc : Display log-scale images instead of linear-scale images
+     -out   : Write ACA data from slots to ASCII files (e.g. -out=1,2,5,6)
+     -page  : Run in display page mode
+     -pea   : PEA unit ('a' or 'b')
+     -raw   : Show raw data 
+     -time  : Minimum time (msec) between screen updates
+     -vcdu  : VCDU count value for start of processing
+     -pause : Start out paused (default=false)
+     -scale : Apply pixel signal scaling: DN = (scale_factor/32 * pixsig - 50) 
+  
+Examples:
+
+Note the use of splat instead of splat.pl.  This is required to grab the
+necessary ACA EGSE perl libraries.
+
+From solaris in ASCDS environment (to get correct GMT):
+
+  /proj/telmon/aca/telem_tools/dump2occ.pl telem.sto.gz > telem.occ
+
+From linux:
+
+  splat -i telem.occ
+
+  gunzip --stdout telem.sto.gz \
+  | $SKA/data/aca_egse/telem_convert/ehs5tdsn.pl \
+  | splat
+
+USAGE
+  ; 
+
+  $ACA_DECOM_HOME = "$ENV{SKA}/decom/aca";
+  $MAX_EVENT_LOG_LINES = 10000;
+  $opt_page = 1;
+  $opt_time = 0;
+  $opt_vcdu = 0;
+  $opt_raw  = 1;
+  $opt_i    = undef;
+  $opt_image= 1;
+  $opt_h    = 0;
+  $opt_tctm = 0;
+  $opt_logsc= 1;
+  $opt_pea  = 'a';
+  $opt_pause= 0;
+  $opt_scale  = 1;
+  
+  &GetOptions ('h',		# Help
+	       'page!',		# Run in display page mode
+	       'image!',	# Display images
+	       'raw!',		# Display raw data
+	       'i=s',		# Optional input file
+	       'time=i',	# Min time between updates
+	       'vcdu=i',	# Minimum VCDU counter value
+	       'tctm!',		# TCTM format?
+	       'logsc!',	# Display log-scale grey-scale images
+	       'out=s',		# Write ACA data from slots to an ASCII file
+	       'pea=s',		# PEA ("a" or "b")
+               'pause!',        # Start out paused
+               'scale!',        # Apply pixel signal scaling: DN = (scale_factor/32 * pixsig - 50)
```

Comment:
I'm not clear on what the scaling is supposed to do.  Do we want it for these data?  Also, should we convert the slot data from DN to e/sec on the Perl side?